mantidimaging/gui/windows/live_viewer/model.py

```diff
@@ -50,11 +51,14 @@ def _remove_oldest_image(self) -> None:
     def use_loading_function(self, func: typing.Callable) -> None:
         self.loading_func = func
 
-    def load_image(self, image: Image_Data) -> np.ndarray:
+    def load_image(self, image: Image_Data) -> np.ndarray | None:
```

Comment:
Does this need to be `np.ndarray | None`, it looks like it either returns the array or raises an exception.